truewiki/views/preview.py

```diff
@@ -7,7 +7,7 @@
 from ..wiki_page import WikiPage
 
 
-def view(user, old_page: str, new_page: str, body: str, summary: str = None) -> web.Response:
+def view(user, old_page: str, new_page: str, body: str, summary: str = None, edit_conflict=False) -> web.Response:
```

Comment:
`edit_conflict` -> `has_edit_conflict`? Makes it a bit more clear it is a boolean value.